client/src/templates/Challenges/utils/build.js

```diff
@@ -13,6 +13,7 @@ import {
 import frameRunnerData from '../../../../../config/client/frame-runner.json';
 // eslint-disable-next-line import/no-unresolved
 import testEvaluatorData from '../../../../../config/client/test-evaluator.json';
+import curriculumHelpers from '../../../utils/curriculum-helpers';
```

Comment:
```suggestion
import { removeJSComments } from '../../../utils/curriculum-helpers';
```
since we're talking about removing `removeJSComments` from the default export